src/css/components/arrange/arrange.css

```diff
@@ -28,7 +28,6 @@
  */
 
```

Comment:
Need to remove the "1" item in the comments above

```css
1. Protect against the component expanding beyond the confines of its
 *    container if properties affecting the box-model are applied to the
 *    component. Mainly necessary because of (5).
 ```

Reorder the numbers in the comments above and for the comments in the CSS.
Since we're taking "1" away, change "2" to "1", "3" to "2" and so on.